doc/OneStepSetup.md

```diff
@@ -64,6 +64,24 @@ A sample conf file is located in the `boot` folder on the SD card.
     # export gotify_domain=https://gotify.domain.com
     # export gotify_app_token=put_your_token_here
     # export gotify_priority=5
+
+    # TeslaUSB can optionally use the Tesla API to keep your car awake, so it can
+    # power the Pi long enough for the archiving process to complete. To enable
+    # that, please provide EITHER your Tesla account email and password, or
+    # an authentication token and refresh token based on those credentials.
+    # To create these tokens, see:
+    # https://tesla-api.timdorr.com/api-basics/authentication
+    # TeslaUSB will only send your credentials to the Tesla API itself.
+    # Note that the token will be immediately refreshed upon first use, which
+    # expires the token. So you shouldn't use a token that any other tool (like
+    # TeslaFi) is currently using.
+    # export tesla_email=joeshmo@gmail.com
+    # export tesla_password=pa$$word
```

Comment:
if you're going to use `pa$$word` as an example password, you should put it in single quotes :)

Reply:
Why? I'm really not sure why the other places in the file suggest single quotes around passwords, either.

Reply:
Oh wait, is it because `$` is a special character in bash? I'll just change it.